<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Woganog</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yamba v1.3 GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="saveFileName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="saveFileText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Save file:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseFileName">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xCoordText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yCoordText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zCoordText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="saveFileText_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="saveFileText_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="saveFileText_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QLabel" name="saveFileText_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Map:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mapText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="goButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Go!</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sunTimeComboBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ignore</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Morning to Midday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Midday to Night</string>
     </property>
    </item>
	<item>
     <property name="text">
      <string>Night to Midnight</string>
     </property>
    </item>
	<item>
     <property name="text">
      <string>Midnight to Morning</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="saveFileText_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sun Time:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeText">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="saveFileText_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time (between 0 and 65535):</string>
    </property>
   </widget>
   <widget class="QLabel" name="woganogText">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Program by Woganog.</string>
    </property>
	<property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mapCodeLink">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Map codes can be found here!</string>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="keepDecryptedCheckBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Keep decrypted save file?</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
